server/src/internalClusterTest/java/org/opensearch/cluster/MinimumMasterNodesIT.java

```diff
@@ -208,7 +208,7 @@ public void testTwoNodesNoMasterBlock() throws Exception {
         otherNode = node1Name.equals(masterNode) ? node2Name : node1Name;
         logger.info("--> add voting config exclusion for master node, to be sure it's not elected");
```

Comment:
There are still a few `master` references on L#209, L#158, L#173

Reply:
Thank you for your careful review!
Ah, I didn't realized there are still lots of log messages contain `master` in `server/src/internalClusterTest` directory and is shown in this PR. 😂
To minimize the change for 2.0 version, I propose to limit the scope of this PR to `server/src/main` directory. For the `master` references in other directories, I will create PR to main branch first and then backport to 2.x branch. Is that fine?

Reply:
The code changes for this PR is actually a subset of https://github.com/opensearch-project/OpenSearch/pull/2519. I didn't plan to backport that PR to 2.0 branch since the feature for 2.0 version is frozen, but when I saw there are still log messages contain `master` terminology, I think it's worth to replace all `master` reference in log message in 2.0 version for consistency.
I will make sure all the log message in `server/src/main` don't have `master` terminology, since it will be exposed to the user.
For the `master` terminology in other directories, I will handle in different PRs ( tracked in issue https://github.com/opensearch-project/OpenSearch/issues/1548)

Reply:
Sounds good to me. 👍

Reply:
Thank you! 😄